front/lib/api/oauth/providers/slack.ts

```diff
@@ -104,14 +103,13 @@ export class SlackOAuthProvider implements BaseOAuthStrategyProvider {
     const clientId = (() => {
       switch (useCase) {
         case "personal_actions":
-        case "platform_actions":
           return config.getOAuthSlackToolsClientId();
         case "connection": {
           return config.getOAuthSlackClientId();
         }
-        case "bot": {
+        case "bot":
+        case "platform_actions":
           return config.getOAuthSlackBotClientId();
```

Comment:
And the reason to use the bot slack app instead of client tool app is that the client tool app doesn't have the scope you need? Because in term of scenario, you're just adding a second Slack tool, and there is nothing really bot specific about it.